app/lib/features/news/actions/submit_news.dart

```diff
@@ -0,0 +1,175 @@
+import 'package:acter/common/providers/sdk_provider.dart';
```

Comment:
Assuming that this is just separate of function code into separate action file, not changes in code logic.

Reply:
yes.